rosdep/python.yaml

```diff
@@ -6399,6 +6399,7 @@ yapf:
     stretch: [yapf]
   ubuntu:
     '*': [yapf]
+    focal: [yapf3]
```

Comment:
So, I feel like this should be a separate key.  That's because in https://github.com/google/yapf/blob/master/README.rst , it says that you should run `yapf` for Python 2 code and `yapf3` for Python 3 code.  So it is clearly a different beast.

We may want to go even further and clean this `yapf` key up.  Since this is only supported up until eoan, we could just remove all of the entries for Ubuntu except for the `xenial` (pip) and `bionic` (debian) entries.  But I won't insist on that, you can do that if you feel like it.

Reply:
Could no follow your last paragraph and deciced to leave `yapf`as is. Added `yapf3` key as you suggested though. Thanks for the hint!